src/App.jsx

```diff
@@ -113,6 +158,10 @@ function App() {
               <center>
```

Comment:
Instead of using the <center> tag, which is deprecated, you can achieve the same result using CSS.

Reply:
Thank you for the feedback. Completed!